src/system/time/services/ntp.cpp

```diff
@@ -1,17 +1,24 @@
 #include "ntp.h"
+#include <NTPClient.h>
+#include <WiFiUdp.h>
+#include "utils/time_converter.cpp"
+
 
 #define NTP_OFFSET   60 * 60         // seconds
 #define NTP_INTERVAL 60 * 1000       // miliseconds
 #define NTP_ADDRESS  "pool.ntp.org"  // URL NTP Server
 
+WiFiUDP ntpUDP;
+NTPClient timeClient(ntpUDP);
```

Comment:
Those variables should be attributes of the class, otherwise each class will reference the same variables.